lib/Common/DataStructures/BaseDictionary.h

```diff
@@ -1239,7 +1239,7 @@ namespace JsUtil
         public:
             EntryIterator(TDictionary &dictionary) : Base(dictionary, 0), entryCount(dictionary.count)
             {
-                if(IsValid() && IsFreeEntry(entries[entryIndex]))
+                if(IsValid() && IsFreeEntry(__super::entries[entryIndex]))
```

Comment:
I recall last time `__super` wasn't supported and we used base class name directly (`Base::` here).